### ISSUE TITLE
IETF and W3C as data standards sources

### ISSUE THREAD

**Issue by ftrotter** [project-open-data/project-open-data.github.io#578]
Hi,
      The data standards bodies that you have listed under your web page on open standards

https://project-open-data.cio.gov/open-standards/

Does not list the IETF or the W3C. This is a problem for several reasons. 

First, those two standards bodies are the closest to actually meeting the letter and spirit of the definition of a "voluntary consensus standards body" according the definition in A-119 https://www.whitehouse.gov/omb/circulars_a119

Second, those are the standards holders for the actual standards that the Internet is using to run APIs and data exchange (REST, JSON, CSV, XML, HTML, HTTP, etc etc)

Third, when I try to explain that an Excel file is not an open standard, but a CSV file IS an open standard, (which by the way, is the single most common violation of the A-119 concept there is) your page on "open standards" does NOTHING to help with that. 

Fourth, organizations like ANSI have [protested the general availability of standards that are published in Federal Regulation](https://share.ansi.org/Shared%20Documents/Standards%20Activities/Critical%20Issues/FederalRegister_IncorporationByReference_IBR/ANSI%20Response%20IBR_041012.docx) (Note that the submitted this comment to further public discourse, in the proprietary Microsoft Word Docx file format.)

In short, you are recommending standards bodies that arguably do not "get it" with regards to open data formats over the standards bodies that obviously do get it, and whose standards are used all of the time for open data efforts.

Which leaves us with this question: Is this an honest oversite that can be fixed with a pull request? I hope so, and as a result, I made one for you. Or is this a a more insidious problem, where the CIOs who are forming this are unwilling for one reason or another to actually make clear who provides real open data formats?

I expect we will know as soon as the pull request is handled...

Regards,
-FT

**Comment by akuckartz** [comment]
:+1:

**Comment by crertel** [comment]
For the curious, the PR in question is #577 .

**Comment by ftrotter** [comment]
hello?

**Comment by TJKoury** [comment]
@ftrotter I hear you loud and clear.  Hopefully this will be fixed, soon.

**Comment by ftrotter** [comment]
Hi. Its been a while. This is an important issue.

Progress?

-FT

**Comment by rebeccawilliams** [comment]
Closing this, thanks again for your patience @ftrotter

**Comment by akuckartz** [comment]
How was this resolved?
